Mathlib/Data/FunLike/Equiv.lean

```diff
@@ -218,7 +218,7 @@ theorem comp_bijective (f : α → β) (e : F) : Function.Bijective (e ∘ f) 
   (EquivLike.bijective e).of_comp_iff' f
 
 /-- This is not an instance to avoid slowing down every single `Subsingleton` typeclass search. -/
-lemma subsingleton_dom [Subsingleton β] : Subsingleton F :=
+lemma subsingleton_dom [FunLike F β γ] [Subsingleton β] : Subsingleton F :=
```

Comment:
This one is weird, since the variable must already be around.

Reply:
Yes, this one was incorrect! Thanks for catching it.

Reply:
It was suggested by the linter due to a bug in 4814 that has since been fixed, so hopefully it won't happen again.